rurusetto/wiki/models.py

```diff
@@ -72,6 +72,8 @@ class Ruleset(models.Model):
     description = models.CharField(default="", max_length=150)
     icon = models.ImageField(default='default_icon.png', upload_to='rulesets_icon', validators=[
         FileExtensionValidator(allowed_extensions=['png', 'gif', 'jpg', 'jpeg', 'bmp', 'svg', 'webp'])])
+    dark_icon = models.ImageField(default='default_icon.png', upload_to='rulesets_icon_dark', validators=[
+        FileExtensionValidator(allowed_extensions=['png', 'gif', 'jpg', 'jpeg', 'bmp', 'svg', 'webp'])])
```

Comment:
```suggestion
    light_icon = models.ImageField(default='default_icon.png', upload_to='rulesets_icon_light', validators=[
        FileExtensionValidator(allowed_extensions=['png', 'gif', 'jpg', 'jpeg', 'bmp', 'svg', 'webp'])])
```
Because the website's default theme is dark so it need to change.